app/views/course/assessment/assessments/_assessment_lesson_plan_item.json.jbuilder

```diff
@@ -1,3 +1,13 @@
 json.partial! 'course/lesson_plan/items/item.json.jbuilder', item: item
 
 json.lesson_plan_item_type @assessment_tabs_titles_hash[item.tab_id]
+json.item_path course_assessment_path(current_course, item)
+folder = @folder_loader.folder_for_assessment(item.id)
+if can?(:attempt, @assessment) && !folder.materials.empty?
+  json.materials do
```

Comment:
I will need to use the `materials` json data - possible to factor that out as a separate partial?